priv/repo/migrations/20190501221754_create_avatar_listings.exs

```diff
@@ -0,0 +1,40 @@
+defmodule Ret.Repo.Migrations.CreateAvatarListings do
+  use Ecto.Migration
+
+  def change do
+    Ret.AvatarListing.State.create_type()
+
+    create table(:avatar_listings, prefix: "ret0", primary_key: false) do
+      add(:avatar_listing_id, :bigint, default: fragment("ret0.next_id()"), primary_key: true)
+      add(:avatar_listing_sid, :string)
+      add(:slug, :string, null: false)
+      add(:order, :integer)
+      add(:state, :avatar_listing_state, null: false, default: "active")
+      add(:tags, :jsonb)
+      add(:avatar_id, :bigint, null: false)
+      timestamps()
```

Comment:
i think more idiomatic to have this at the end

Reply:
Was separating fields that are conceptually part of the listing and fields that are just cloned from the avatar record the listing is for. Probably not a super important distinction.